### PR TITLE
Support for xCode7 simulator

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: lib/iphone-simulator-xcode-6.ts [hunk 1]
```diff
@@ -12,89 +14,131 @@ export class XCode6Simulator implements ISimulator {
...
-			var selectedDevice = availableDevices[fullDeviceIdentifier];
-			if(selectedDevice) {
-				return selectedDevice[0].device;
+	private launch(udid: string, bundleId: string): IFuture<void> {
+		return this.simctlExec('launch', [udid, bundleId]);
+	}
+
+	private getAvailableDevices(): IFuture<IDevice[]> {
+		return (() => {
+			var rowDevices = this.simctlExec("list", ["devices"]).wait();
```
Comment (teobugslayer) [inline lib/iphone-simulator-xcode-6.ts]: Is this "row" or "raw"?

File: lib/iphone-simulator-xcode-6.ts [hunk 2]
```diff
@@ -12,89 +14,131 @@ export class XCode6Simulator implements ISimulator {
...
+
+			console.log("ROW DEVICESS!!!");
```
Comment (teobugslayer) [inline lib/iphone-simulator-xcode-6.ts]: Is this a debugging code left incidenetaly?

**Comment by Fatme** [comment]
I'm closing this PR for now due to some git issues.

File: lib/iphone-simulator-xcode-6.ts [hunk 3]
```diff
@@ -12,89 +14,131 @@ export class XCode6Simulator implements ISimulator {
...
+	private buildFullDeviceIdentifier(deviceName: string): string {
+		return util.format("%s.%s", XCode6Simulator.DEVICE_IDENTIFIER_PREFIX, deviceName);
+	}
 
-					if(!this.availableDevices[deviceIdentifier]) {
-						this.availableDevices[deviceIdentifier] = [];
-					}
+	private simctlExec(command: string, args: string[]): IFuture<any> {
+		args = ["xcrun", "simctl", command, ...args];
+		console.log(util.format("Executing: xcrun with args: %s", args.join(' ')));
```
Comment (teobugslayer) [inline lib/iphone-simulator-xcode-6.ts]: Is this debugging code left?

**Pull request closed without merging**